Formula/termtosvg.rb

```diff
@@ -0,0 +1,23 @@
+class Termtosvg < Formula
+  include Language::Python::Virtualenv
+
+  desc "Record terminal sessions as SVG animations"
+  homepage "https://nbedos.github.io/termtosvg"
+  url "https://github.com/nbedos/termtosvg/archive/0.6.0.tar.gz"
+  sha256 "09ccefb75a6a4f8be186b6efdc3cf355cf8de7dc2222b9e4d841d54e623dd261"
+
+  depends_on "python"
+
+  def install
+    venv = virtualenv_create(libexec, "python3")
+    system libexec/"bin/pip", "install", "--upgrade", "setuptools"
```

Comment:
Why `pip install --upgrade`?

Reply:
okay… I removed that one…

Reply:
That's not what I meant. Why `pip install pip` and `setuptools` at all? They are provided by the python formula.